api-catalog-services/src/main/resources/apicatalog-log-messages.yml

```diff
@@ -39,6 +39,13 @@ messages:
       reason: "The status of one or more containers could not be retrieved."
       action: "Check the status of the message for more information and the health of the Discovery Service."
 
+    - key: org.zowe.apiml.apicatalog.apiDocHostCommunication
+      number: ZWEAC105
+      type: WARNING
+      text: "API Documentation not retrieved for service '%s' due to communication error, %s"
+      reason: "Unable to fetch API documentation."
+      action: "Make sure the service documentation url or server transport encoding is configured correctly."
+
```

Comment:
This comment is just a reminder to verify after merge that this message is reflected in the docs. It should be an automated PR to docs-site/ but it needs to be followed to make sure someone reviews it and merges it there too.

Reply:
ok, i have made a note